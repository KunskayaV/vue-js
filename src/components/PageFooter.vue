<template>
   <footer class="page-footer">
      <img alt="Footer logo" class="logo" src="@/assets/logo.png" width="180" height="25" />
    </footer>
</template>

<style scoped>
.page-footer {
  display: flex;
  justify-content: center;
  padding-block: 24px;
  background-color: #555555;
}
</style>