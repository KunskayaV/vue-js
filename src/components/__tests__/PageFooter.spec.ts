import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import PageFooter from '../PageFooter.vue'

describe('PageFooter', () => {
  it('renders properly', () => {
    const wrapper = mount(PageFooter)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
