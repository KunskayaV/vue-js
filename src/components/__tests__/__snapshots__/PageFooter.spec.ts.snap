// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`PageFooter > renders properly 1`] = `"<footer data-v-8ee5c97c=\\"\\" class=\\"page-footer\\"><img data-v-8ee5c97c=\\"\\" alt=\\"Footer logo\\" class=\\"logo\\" src=\\"/src/assets/logo.png\\" width=\\"180\\" height=\\"25\\"></footer>"`;
